packages/blade/src/components/Checkbox/__tests__/__snapshots__/Checkbox.web.test.tsx.snap

```diff
@@ -89,7 +89,7 @@ exports[`<Checkbox /> should render checkbox with label 1`] = `
 }
 
 .c10.c10.c10.c10.c10 {
-  color: hsla(227,100%,59%,1);
+  color: hsla(227,71%,51%,1);
```

Comment:
Why is it updating Checkbox, collapsible, Link and List snapshots?

Reply:
There were some token updates that happened, possibility because of that. 

https://razorpay.slack.com/archives/G01B3LQ9H0W/p1704953467716609